### PR TITLE
Design alignment updates

### PR DESCRIPTION
**Pull request by marcellamaki**
## Summary

Handles various pieces of design speccing alignment feedback, including some tasks in #8563 and other PR feedback and issues.

## Included Items

Topics Page
- [x]  Add "active" styles for folders and search tab
- [x] Active side panel buttons styles
Screen Shot 2021-11-04 at 7 49 25 PM

Library page
- [x]  Fix mobile card view that sometimes shows up as a grid with toggle, when should just be single card
- [x] no toggle on 0 results (in case there are ever a 0 results page)

Screen Shot 2021-11-04 at 7 47 29 PM 

- [x] Active side panel buttons styles

|  view 1  |  view 2 |
|---|---|
| Screen Shot 2021-11-04 at 7 31 20 PM | Screen Shot 2021-11-04 at 7 31 08 PM |

Bookmarks page
- [x]  Card: Add text truncation to manage long description overflow and spacing issues

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by jtamiace — APPROVED** [review]

File: kolibri/plugins/learn/assets/src/views/LibraryPage.vue [hunk 1]
```diff
@@ -308,13 +310,21 @@
...
-        } else if (this.windowBreakpoint < 2) {
+          return null;
+        } else if (this.windowIsSmall) {
           return 2;
         } else {
           return 3;
         }
       },
+      activeActivityButtons() {
+        console.log(this.searchTerms.learning_activities);
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/views/LibraryPage.vue]:
```suggestion
```

File: kolibri/plugins/learn/assets/src/views/LibraryPage.vue [hunk 2]
```diff
@@ -308,13 +310,21 @@
...
+        return this.searchTerms.learning_activities;
+      },
+      activeCategories() {
+        console.log(this.searchTerms.categories);
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/views/LibraryPage.vue]:
```suggestion
```

File: kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue [hunk 3]
```diff
@@ -55,7 +57,10 @@
             :text="coreString(camelCase(category))"
             appearance="flat-button"
             :appearanceOverrides="customCategoryStyles"
-            :disabled="availableRootCategories && !availableRootCategories[val]"
+            :disabled="availableRootCategories &&
+              !availableRootCategories[val] &&
+              !activeKeys.filter(k => k.includes(val)).length"
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue:70]: Should do this as a computed prop - and there's probably a better way to do the existence check.

**Review by rtibbles — APPROVED** [review]
Let's merge this - but I think we can do some more cleanup here.

**Pull request merged**